test/my-input.html

```diff
@@ -0,0 +1,34 @@
+<!--
+  This custom element is needed for testing the "attr-for-value" feature of
+  <vaadin-combo-box-light>
+-->
+<link rel="import" href="../../polymer/polymer-element.html">
+<dom-module id="my-input">
+  <template>
+    <style>
+      :host {
+        display: inline-block;
+      }
+    </style>
+    <iron-input id="input" bind-value="{{customValue}}">
```

Comment:
Missing import for `<iron-input>` component dependency here.

Reply:
Added, thanks.